### DISCUSSION TITLE
[remote_config] Support for Flutter Web

### DISCUSSION THREAD

**Discussion by red010b37** [firebase/flutterfire#5593] [Feature request]
Currently there is no support for Flutter Web

Remote Config is a Key firebase feature that should be cross platform supported.

**Comment by Viacheslav-Romanov** [comment]
Can you assign someone on this, guys? Thanks.

**Comment by Ehesp** [comment]
This is planned, but is not currently being worked on. We'll update the roadmap #2582 with more information when applicable.

**Comment by tungtrinh** [comment]
> This is planned, but is not currently being worked on. We'll update the roadmap #2582 with more information when applicable.

Could you please give us update on this. As I can see flutter fire implementation for web is making huge progress but this has no mention at all.

**Reply by bounty1342** [reply]
Yes, can we have an ETA, so we can plan accordingly or find alternatives.

**Reply by Ehesp** [reply]
There's currently no fixed ETA on this implementation, although I'd wager a guess it'll be around August (which, is a total guess).

**Reply by Salakar** [reply]
PRs also welcome in the meantime since the plugin is already federated so it's just a case of following the platform interface API and implementing a web package

**Comment by PlugFox** [comment]
@red010b37 @Viacheslav-Romanov @tungtrinh 

Checkout my implementation:
https://gist.github.com/PlugFox/9e6b16890deacd0ae4855dae3491a149

**Reply by IchordeDionysos** [reply]
Nice work 😍 

Would be great to get that officially into the flutterfire project 💪

**Reply by doppio** [reply]
@PlugFox Have you considered submitting a pull request for those changes?

**Reply by PlugFox** [reply]
@doppio 
This is only a partial implementation, needed for my tasks.
All the basic functionality is there.

But I will not do a pull request in the near future, I apologize.

If you want, you can use my code at your own discretion without specifying the author.
